pkg/booleanpolicy/validate.go

```diff
@@ -48,9 +47,6 @@ func Validate(p *storage.Policy, options ...ValidateOption) error {
 	}
 
 	errorList := errorhelpers.NewErrorList("policy validation")
-	if !policyversion.IsBooleanPolicy(p) {
```

Comment:
I'd check `policy_version` is "1.1", assuming you plan to keep this proto field.